### ISSUE TITLE
This PDF doesn't load

### ISSUE THREAD

**Issue by mguidoti** [gsautter/goldengate-imagine#856]
Hi Guido,

We bumped into a PDF that simply didn't load in two of our machines here. Remember that we already set the memory allocation a bit higher, and still, this imposes a challenge. i tested in the machines with the highest and lowest configs here.

Here's the [link to the PDF](https://drive.google.com/open?id=1IzsTc-AeZCUgjhW6dxUHNhjfDCdtHLmp), if you want to take a look. 

Also, the two error log files are attached here. They're essentially the same.. 

[GgImagine.20200117-0854.err.log](https://github.com/gsautter/goldengate-imagine/files/4077416/GgImagine.20200117-0854.err.log)
[GgImagine.20200117-1205.err.log](https://github.com/gsautter/goldengate-imagine/files/4077417/GgImagine.20200117-1205.err.log)

I felt that this was important to report.

Best and thanks!

**Comment by gsautter** [comment]
PDFs that fail to open are surely important to report ... and both logs in fact show the same error.

However, I can open it just fine here on my machine, most likely because the error is the very same that @myrmoteras reported in https://github.com/gsautter/goldengate-imagine/issues/789 and that I have fixed since ... comes with next update.

**Comment by myrmoteras** [comment]
I can open it on my machine already

**Comment by gsautter** [comment]
OK then ... maybe the other machines are behind on updates. If there is anything else going on, it must be really strange, e.g. fonts behaving differently on different machines, as memory should not be an issue at all in a PDF this size.

**Comment by mguidoti** [comment]
That's how their GGI is looking like:

![image](https://user-images.githubusercontent.com/44419136/72634292-06e7e000-3939-11ea-9e04-3c231e42aee5.png)

Is there any other update that we're still missing?

**Comment by gsautter** [comment]
This is not about the configuration, but the updates for core GGI ... the PDF decoder is part of the latter. Please check the `Update` folder below GGI root and check if there is a file named `GgUpdate.IM.20191219-0100.zip.done`. If not, check the `UpdateHosts.cnfg` file and see whether or not there is a line reading `http://tb.plazi.org/GgServer/Updates`, and add it if there isn't. Afterwards, restart GGI and allow web access so the starter can download and install the updates.

**Comment by mguidoti** [comment]
Yes, we added the `http://tb.plazi.org/GgServer/Updates` on the `UpdateHosts.cnfg` last year, remember?

And the file ending on `0100` is there, last modified on Dec 19.

**Comment by gsautter** [comment]
OK, then the updates _should_ be installed ... what size (in bytes) is `ImageMarkupPDF.jar`, and what time was it modified last?

**Comment by mguidoti** [comment]
There you go:

![image](https://user-images.githubusercontent.com/44419136/72635775-61cf0680-393c-11ea-808b-8421f587847f.png)

**Comment by gsautter** [comment]
OK, that looks up to date ... hard to tell why the PDF fails to open anyway.

Is there any differences between your machine (where it does open) and the ones where it does not? JAR file sizes, modification dates, etc.?

**Comment by mguidoti** [comment]
I haven't tried on my personal notebook, because it's the slowest machine in the office... Just in two of their machines... 

Let me try.

**Comment by gsautter** [comment]
Sorry, I was mistaking ... was @myrmoteras stating he could open it.

In any case, the `...out.log` might also be helpful.

**Comment by mguidoti** [comment]
Ok, in that case, I'll ask them to re-open the document so the `...out.log` can be updated with this info again (they were working on other things this whole time).

Just a moment.

**Comment by mguidoti** [comment]
Ok, they reproduced the problem and here are the two ...out.log (I haven't look into it):

[GgImagine.20200117-1205.out.log](https://github.com/gsautter/goldengate-imagine/files/4078466/GgImagine.20200117-1205.out.log)
[GgImagine.20200117-1550.out.log](https://github.com/gsautter/goldengate-imagine/files/4078467/GgImagine.20200117-1550.out.log)

**Comment by gsautter** [comment]
Hard to tell, the logs look pretty normal to me ... not sure how to proceed here ... I'll try and make a new build this weekend, so how about getting back to this on Monday? Right now, the logs from your machines don't match my code, so it's hard to follow the logged stack traces to where the error occurs, and that will change with a new build (until I change anything further on my end, that is ... but that's always the case).

**Comment by mguidoti** [comment]
Absolutely!

Thanks a lot for looking at this.

**Comment by gsautter** [comment]
Let's see how this behaves with the new build I just put online ...

**Comment by mguidoti** [comment]
Hi @gsautter, 

This worked in both computers. Thanks a lot.
